packages/shared/src/components/sidebar/common.tsx

```diff
@@ -146,6 +146,7 @@ export const ItemInner = ({
           shouldShowLabel ? 'opacity-100 delay-150' : 'opacity-0',
           item?.className?.text,
         )}
+        title={item.title}
```

Comment:
This was just missing?

Reply:
Yeah, to get the popover (native in browser)